mlflow/server/js/src/components/MetricView.js

```diff
@@ -79,14 +98,15 @@ class MetricView extends Component {
               margin={{top: 10, right: 10, left: 10, bottom: 10}}
             >
               <XAxis dataKey="index" type="number"/>
-              <Tooltip isAnimationActive={false} labelStyle={{display: "none"}}/>
+              <Tooltip isAnimationActive={false} labelStyle={{display: "none"}} />
               <CartesianGrid strokeDasharray="3 3"/>
-              <Legend verticalAlign="bottom"/>
+              <Legend verticalAlign="bottom" payload={this.getLegendPayload('line')}/>
```

Comment:
This was the one component for which I ended up writing custom truncation logic, since there's no API for specifically customizing the style of each legend item (direct links to API docs on the recharts website are weirdly broken, but see http://recharts.org/ --> Api --> Legend). An alternative would be overriding the `content` property of `Legend`, but that seemed a bit heavyweight / I didn't want to reimplement rendering the Legend's contents for MLflow (I'd basically have to reimplement some of [this logic](https://github.com/recharts/recharts/blob/master/src/component/DefaultLegendContent.js))